src/data-pipeline/etl-common/src/test/resources/sensors-data/expected/test_sensors_data_parseLineToDBRow_event.json

```diff
@@ -0,0 +1,183 @@
+  "page_view_entrances" : null,
+  "app_start_is_first_time" : null,
+  "upgrade_previous_app_version" : null,
+  "upgrade_previous_os_version" : null,
+  "search_key" : null,
+  "search_term" : null,
+  "outbound_link_classes" : null,
+  "outbound_link_domain" : null,
+  "outbound_link_id" : null,
+  "outbound_link_url" : null,
+  "outbound_link" : null,
+  "user_engagement_time_msec" : 0,
+  "user_id" : "9113946abd821d52",
+  "user_pseudo_id" : "9113946abd821d52",
+  "session_id" : null,
+  "session_start_time_msec" : 1689819428756,
+  "session_duration" : null,
+  "session_number" : null,
+  "scroll_engagement_time_msec" : null,
+  "sdk_error_code" : null,
+  "sdk_error_message" : null,
+  "sdk_version" : null,
+  "sdk_name" : null,
+  "app_exception_message" : null,
+  "app_exception_stack" : null,
+  "custom_parameters" : {
+    "x-ga-system_properties" : {
+      "is_login_id" : false,
+      "ip" : null,
+      "province" : null,
+      "city" : null,
+      "user_agent" : null,
+      "url" : null,
+      "screen_name" : null,
+      "title" : null,
+      "lib_method" : null,
+      "is_first_day" : false,
+      "os" : null,
+      "os_version" : null,
+      "lib" : null,
+      "manufacturer" : null,
+      "model" : null,
+      "brand" : null,
+      "app_version" : null,
+      "screen_width" : 0,
+      "screen_height" : 0,
+      "carrier" : null,
+      "timezone_offset" : 0,
+      "app_id" : null,
+      "app_name" : null,
+      "wifi" : false,
+      "network_type" : null,
+      "lib_plugin_version" : null,
+      "device_id" : null,
+      "x_test_user" : null,
+      "product_id" : 0,
+      "product_name" : null,
+      "product_classify" : null,
+      "product_price" : 0.0,
+      "item_price" : 0.0,
+      "event_duration" : 0,
+      "referrer" : null,
+      "referrer_title" : null,
+      "resume_from_background" : false,
+      "is_first_time" : false,
+      "age" : 0,
+      "name" : null,
+      "user_id" : null,
+      "user_name" : null,
+      "browser" : null,
+      "browser_language" : null,
+      "browser_version" : null,
+      "$lib_method" : "autoTrack",
+      "$carrier" : "T-Mobile",
+      "$os_version" : "12",
+      "$is_first_day" : true,
+      "$model" : "sdk_gphone64_arm64",
+      "$os" : "Android",
+      "$screen_width" : 1080,
+      "$brand" : "GOOGLE",
+      "$wifi" : true,
+      "$network_type" : "WIFI",
+      "$screen_height" : 2400,
+      "$app_version" : "3.0.3",
+      "$lib" : "Android",
+      "$device_id" : "9113946abd821d52",
+      "$app_name" : "Notepad",
+      "$lib_plugin_version" : [ "android:3.5.3" ],
+      "$title" : "Notepad",
+      "$lib_version" : "6.7.2",
+      "$timezone_offset" : -480,
+      "$app_id" : "com.farmerbb.notepad",
+      "$url" : "com.farmerbb.notepad.android.NotepadActivity",
+      "$screen_name" : "com.farmerbb.notepad.android.NotepadActivity",
+      "$manufacturer" : "GOOGLE"
+    },
+    "screen_resolution" : "0x0",
```

Comment:
"screen_resolution" : "0x0",  need to parse and  set to screen_height/width

Reply:
It has been fixed in the properties model